config/src/lib.rs

```diff
@@ -901,6 +903,95 @@ impl Config {
 
         Ok(())
     }
+
+    /// List the data in the foundry cache
+    pub fn list_foundry_cache() -> eyre::Result<Cache> {
+        if let Some(cache_dir) = Config::foundry_cache_dir() {
+            let mut cache = Cache { chains: vec![] };
```

Comment:
let's derive Default and make this `Cache::default`

Reply:
@mattsse putting this code in default, prevents us from wrapping it in `Result` afaik, just returning an empty Cache struct doesn't seem great, can you elaborate ?